ostree/ostree_transport.go

```diff
@@ -0,0 +1,171 @@
+package ostree
```

Comment:
I pushed a new version.  I changed the repo/image hierarchy separator to be ':' as I found '@' to be confusing (we already use @ for "image@repo")